mixer/adapter/solarwinds/config/config.proto

```diff
@@ -24,6 +24,67 @@ option (gogoproto.goproto_getters_all) = false;
 option (gogoproto.equal_all) = false;
 option (gogoproto.gostring_all) = false;
 
+// Example config usage
+// apiVersion: "config.istio.io/v1alpha2"
+// kind: solarwinds
+// metadata:
+//   name: handler
+//   namespace: istio-system
+// spec:
+//   appoptics_access_token: <APPOPTICS SAMPLE TOKEN>
+//   papertrail_url: <PAPERTRAIL URL>
+//   papertrail_local_retention_duration: <RETENTION PERIOD FOR LOGS LOCALLY, Optional>
+//   metrics:
+//     requestcount.metric.istio-system:
+//       name: request_count
+//       label_names:
+//       - source_service
+//       - source_version
+//       - destination_service
+//       - destination_version
+//       - response_code
+//     requestduration.metric.istio-system:
+//       name: request_duration
+//       label_names:
+//       - source_service
+//       - source_version
+//       - destination_service
+//       - destination_version
+//       - response_code
+//     requestsize.metric.istio-system:
+//       name: request_size
+//       label_names:
+//       - source_service
+//       - source_version
+//       - destination_service
+//       - destination_version
+//       - response_code
+//     responsesize.metric.istio-system:
+//       name: response_size
+//       label_names:
+//       - source_service
+//       - source_version
+//       - destination_service
+//       - destination_version
+//       - response_code
+//     tcpbytesent.metric.istio-system:
+//       name: tcp_bytes_sent
+//       label_names:
+//       - source_service
+//       - source_version
+//       - destination_service
+//       - destination_version
+//     tcpbytereceived.metric.istio-system:
+//       name: tcp_bytes_received
+//       label_names:
+//       - source_service
+//       - source_version
+//       - destination_service
+//       - destination_version
+//   logs:
+//     solarwindslogentry.logentry.istio-system:
+//       payloadTemplate: '{{or (.originIp) "-"}} - {{or (.sourceUser) "-"}} [{{or (.timestamp.Format "2006-01-02T15:04:05Z07:00") "-"}}] "{{or (.method) "-"}} {{or (.url) "-"}} {{or (.protocol) "-"}}" {{or (.responseCode) "-"}} {{or (.responseSize) "-"}}'
+
```

Comment:
Please remove the blank line so that the comment is associated with the "message Params" element.

Also, wrap the yaml block in <backquote><backquote><backquote>yaml so that it'll render as YAML in our docs.